File: Python/Search/LinearSearch.py
def search(arr, N, target):
    size = len(arr)

    for i in range(0, size):
        if arr[i] == target:
            return i
    return -1
    
if __name__ == "__main__": 
    arr = [2, 3, 4, 10, 40] 
    x = 10
    N = len(arr) 
  
    result = search(arr, N, x) 
    if(result == -1): 
        print("Element is not present in array") 
    else: 
        print("Element is present at index", result) 

File: Java/Search/LinearSearch.java
package Java.Search;

class LinearSearch {
    static int search(int[] arr, int size, int number) {
        for (int i = 0; i < size; i++) {
            if (arr[i] == number) {
                return i;
            }
        }
        return -1;
    }

    public static void main(String[] args){
        int[] arr = {2, 3, 4, 5, 6, 9, 11};
        int number = 11;

        int result = search(arr, arr.length, number);

        if(result == -1){
            System.out.println(number + " Number is not found");
        } else{
            System.out.println("Number is found at index: " + result);
        }
    }
}